WINC-1092: [release-4.10][docs] Drop AWS support in WMCO 5.x and lower (#1730)
openshift/windows-machine-config-operator: docs/wmco-prerequisites.md

## Supported Cloud Providers based on OKD/OCP Version and WMCO version
| Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
|---------------------------|---------------------------|------------------------|
| VMware vSphere            | 4.7+                      | WMCO 2.0+              |
| Platform none (BYOH)      | 4.8+                      | WMCO 3.1.0+            |

> Note: We added Azure and AWS support in 4.6 but given that Microsoft and AWS have [stopped publishing Windows Server 2019 images with

Reviewer: ```suggestion
Note: We added Azure and AWS support in 4.6, but because Microsoft and AWS have [stopped publishing Windows Server 2019 images with
```

Docker](https://techcommunity.microsoft.com/t5/containers/important-update-deprecation-of-docker-virtual-machine-images/ba-p/3646272),

> we have decided to drop Azure and AWS support for releases older than 6.0.0. For 5.y.z and below it was a requirement for

Reviewer: ```suggestion
we have decided to drop Azure and AWS support for releases older than 6.0.0. For 5.y.z and older, it was a requirement for
```

the Windows Server 2019 images to have Docker pre-installed. From 6.0.0 onwards we are using containerd as the
runtime and it is WMCO's responsibility to manage that.

> 

Reviewer: Quick note: It's unclear what the "4.6" should be attributed to. Is that a version number for Azure, AWS, or WMCO? Also, I'm unsure how relaxed vs. how professional you want this to be. Generally, though, I would suggest that you remove the "we" in these statements. Just one example: 
_

> Before this update, WMCO 4.6 supported Azure and AWS. However, because Microsoft and AWS no longer publish Windows Server 2019 images with Docker, releases older than WMCO 6.0.0 no longer support Azure. For WMCO 5.y.z and older, Windows Server 2019 images were required to have Docker pre-installed. From WMCO 6.0.0 and onward, containerd will serve as the runtime and will be managed by WMCO.

_
